goscord/gateway/event/guild_role_delete.go

```diff
@@ -0,0 +1,24 @@
+package event
+
+import (
+	"github.com/Goscord/goscord/goscord/discord"
+	"github.com/Goscord/goscord/goscord/rest"
+	"github.com/goccy/go-json"
+)
+
+// GuildRoleDelete Is sent when a guild role is created.
```

Comment:
You are sure what "GuildRole**Delete** Is sent when a guild role is **created**." ? 👀

Reply:
ah oops mb, ima fix that